src/components/ListBox/ListBoxMenuItem.js

```diff
@@ -25,7 +26,16 @@ const ListBoxMenuItem = ({ children, isActive, isHighlighted, ...rest }) => {
   });
   return (
     <div className={className} {...rest}>
-      {children}
+      {componentsX ? (
+        <div
+          className={`${prefix}--list-box__menu-item__option`}
```

Comment:
Do we need keyboard handlers if this is `role="button"`? Or potentially another interactive prop like `onClick`?

Reply:
I believe that behavior is already handled by Downshift, these attributes are just holdovers from vanilla and can probably be removed altogether